src/OpenTelemetry.Instrumentation.Hangfire/CHANGELOG.md

```diff
@@ -2,12 +2,13 @@
 
 ## Unreleased
 
+## 1.0.0-beta.2
```

Comment:
Add the date: https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/main/CONTRIBUTING.md#how-to-request-for-release-of-package